src/gluonts/model/seq2seq/_forking_network.py

```diff
@@ -77,7 +77,7 @@ def __init__(
         embedding_dimension: List[int],
         distr_output: Optional[DistributionOutput] = None,
         quantile_output: Optional[QuantileOutput] = None,
-        scaling: bool = False,
+        scaling: bool = True,
```

Comment:
Should we update the default for `scaling` back to `False` here?

Reply:
Right!